website/source/docs/post-processors/checksum.html.md

```diff
@@ -0,0 +1,43 @@
+---
+description: |
+    The checksum post-processor computes specified checksum for the artifact list
+    from an upstream builder or post-processor. All downstream post-processors will
+    see the new artifacts. The primary use-case is compute checksum for artifacts
+    allows to verify it later.
+layout: docs
+page_title: 'Checksum Post-Processor'
```

Comment:
Great thanks for adding docs! Also make sure to add this to the sidebar. You can preview the site with:

```
cd website
make dev
```

To make sure things are formatted correctly and navigation is working.